src/resources/asset.rs

```diff
@@ -82,14 +82,7 @@ impl Asset {
             return Err(Error::AssetFile(absolute_location));
         }
         // Use filename as embedded file path with content from absolute_location.
-        let binding = utils::normalize_path(Path::new(link));
-        debug!("Extracting file name from = {:?}, binding = '{binding:?}'", &full_filename.display());
-        let filename = if cfg!(target_os = "windows") {
-            binding.as_os_str().to_os_string()
-                .into_string().expect("Error getting filename for Local Asset").replace('\\', "/")
```

Comment:
I'm not sure if that is good to remove that code. I remember there were problems on Win, so I added that piece of code. I doubt that is good.

Reply:
Any exact cases could be added testcases?

Things like https://learn.microsoft.com/en-us/windows/win32/fileio/naming-a-file?

Reply:
That code is related to problem of building epub book on Windows platform. We should change window's based '\\' to path like '/' as that should be by epub specification.

See also 
`#[test] fn test_normalize_path_win() {`

https://github.com/Michael-F-Bryan/mdbook-epub/issues/11

Reply:
I think epub-builder already do its best [here to not use backspaces in zip files](https://github.com/crowdagger/epub-builder/blob/56a783e3826fb0a0d335d5a5070e398ecfb5072a/src/zip_library.rs#L62-L63) to conform ePub spec. And I check the book generated with this PR works fine with calibre ebook-viewer that images are embeded and displayed without error.
We need to normalize but not canonicalize the path causing issue. So just using `strip_prefix` is enough.

Reply:
BTW, the `link` parsed by pulldown-cmark here should be a valid [link text](https://spec.commonmark.org/0.30/#images) according to [CommonMark Spec](https://commonmark.org/). It's wrong to accept a backslash-separated path or test it since pulldown-cmark crate is a pull parser for [CommonMark](http://commonmark.org/).

Reply:
`binding` may give wrong path to add assets to zip. Consider the link in test cases at L98, `../upper/assets/asset1.jpg`, is the image added to zip with `upper/assets/asset1.jpg`?

Reply:
About normalizing path, we could use `normpath` instead [like mdbook-pandoc does](https://github.com/max-heller/mdbook-pandoc/commit/319c7bab) later.